### ISSUE TITLE
Change zh_HK to yue_HK in languages.py

### ISSUE THREAD

**Issue by Eletroboss** [AwesomeTTS/awesometts-anki-addon#322]
In short: Google's text-to-speech services do not recognise `zh_HK` for Cantonese but do recognise `yue_HK`.

In long:

`zh_HK` as far as I know refers to 書面語 which is the Hong Kong variant of Standard Written Chinese using Cantonese pronunciation.

It is distinct from Hong Kong Cantonese, which is called 粵語 and I believe uses `yue_HK`.

Both use Cantonese pronuncation, so the distinction is unnecessary except that **in Android with Google's speech services' Cantonese pronunciation it is necessary that it is yue_HK** else it will not be able to find a compatible text-to-speech generator.

What I do to get around this is I go into awesometts/service/languages.py and change zh_HK to yue_HK and it works.

I suggest that zh_HK be replaced with yue_HK, or even better that both are "merged" so to speak so that both codes are tried before giving up on finding a voice.

**Comment by luc-vocab** [comment]
thanks for raising this, I want to confirm again, zh_HK in a realtime TTS tag does not produce any text to speech output on AnkiDroid?

**Comment by Eletroboss** [comment]
Yes - at least with Google speech services on LineageOS and Motorola. It says there is no sound engine for zh_HK.
